### DISCUSSION TITLE
ConvRNNCells dynamic input_shape

### DISCUSSION THREAD

**Discussion by HaydenFaulkner** [apache/incubator-mxnet#16296] [Q&A] [answered]
Same question as the closed #8848, is there a way to dynamically set the input shape for ConvLSTMCell and ConvGRUCell?

Thanks,
Hayden

**Comment by lanking520** [comment]
Usually if you are not sure about the shape of a certain dimension, you can set it to 0. 
@haojin2 if you know anyone can answer this

**Comment by szha** [comment, marked as answer]
This is not supported.
